### ISSUE TITLE
Renderer plugin not working

### ISSUE THREAD

**Issue by antoineFrau** [remarkjs/react-markdown#474]
### Subject of the issue

Hi, I tried to implement remark-math plugin in with react-markdown (https://github.com/antoineFrau/react-markdown-math) to be trigger on Latex and render a MathJax Node.

But of what I saw the plugin is never trigger. I don't know if it's an issue with the remark-math plugin or with react-markdown.

```js
const source= `Lift($L$) can be determined by Lift Coefficient ($C_L$) like the following
  equation.
  
  $$
  L = \frac{1}{2} \rho v^2 S C_L
  $$`;
...
 <ReactMarkdown 
                source={source}
                plugins={[RemarkMathPlugin]}
                renderers={{
                    math: ({ value }) => {
                        console.log("math equation")
                        return <Node>{value}</Node>
                    },
                    inlineMath: ({ value }) =>{
                        console.log("inlineMath equation")
                        return <Node inline={true}>{value}</Node>
                    }
                }}
/>
```

### Your environment

*   **OS**: MacOS 
*   **Packages**:     "react-markdown": "^4.3.1",
    "react-mathjax": "^1.0.1",
    "remark-math": "^4.0.0"
*   **Env**: Firefox 81.1, yarn 1.22

### Steps to reproduce

I pushed a create-react-app with the needed code to test this issue.
- https://github.com/antoineFrau/react-markdown-math

### Expected behavior

So it should print the console.log() because I have math equation Latex format in my code.
#357 

### Actual behavior

Doing nothing. Rendering math equation into TextRenderer.
![image](https://user-images.githubusercontent.com/18289388/96093493-80166280-0ecc-11eb-9bb2-3c5b99705330.png)

**Comment by antoineFrau** [comment]
I just saw that's there is an issue with plugins : #188

**Comment by wooorm** [comment]
Correct. And see https://github.com/remarkjs/react-markdown/issues/470

**Comment by antoineFrau** [comment]
>The plan earlier is explained here: #470 (comment).
It seems this PR would be extremely incompatible with the current project. So the plan was instead to mostly bump remark and fix plugins (#425) first. Then do this PR later.

I saw that later my bad.
Do we have a date for the plugin fix ? Do you guys need help on something ?

**Comment by wooorm** [comment]
> We’ll cut a major version soon, with the goal of fixing a bunch of bugs, but otherwise not changing too much.

No date other than that. For help: there are a lot of open issues that could use some help!

**Comment by wooorm** [comment]
Fixed in 5.0.0!
